.changeset/wild-pigs-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs': patch
+---
+
+add fetch resolver to use an url with \$ref
```

Comment:
Updated the rendering of `AsyncApi` definitions to be able to resolve absolute HTTP `$ref` references.

Reply:
done 😉